### PR TITLE
Add pytorch to inference model

### PR DESCRIPTION
**Pull request by ManfeiBai**
Add pytorch to inference model

### CODE REVIEW

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala [hunk 1]
```diff
@@ -67,10 +96,34 @@ class PyTorchModelSpec extends FunSuite with Matchers with BeforeAndAfterAll
...
+          println(model)
+          model shouldNot be(null)
+
+          model2.doLoadPyTorch(modelBytes)
+          println(model2)
+          model2 shouldNot be(null)
+        }
+      }
+    })
+    threads.foreach(_.start())
```
Comment (qiuxin2012) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala:113]: The exception happened in Thread won't be throw to parent Thread, so the assert failure and exception won't be catched. Please try to use `Thread.UncaughtExceptionHandler` or `Future` to run multithread.

Comment (ManfeiBai) [reply zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala:120]: Thanks！

**Comment by qiuxin2012** [comment]
@ManfeiBai  Please update the document https://analytics-zoo.github.io/master/#APIGuide/PipelineAPI/net/#load-torch-model

**Comment by qiuxin2012** [comment]
https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/APIGuide/PipelineAPI/inference.md We should add the pytorch usage document here.

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala [hunk 2]
```diff
@@ -16,49 +16,84 @@
...
-  val modelURL = s"$s3DataUrl/pytorch/pytorch-resnet.pt"
-  var tmpDir: File = _
   var model: InferenceModel = _
   var model2: InferenceModel = _
   val currentNum = 10
   var modelPath: String = _
+  var modelPathMulti: String = _
+  val h: UncaughtExceptionHandler = new UncaughtExceptionHandler {
+    override def uncaughtException(t: Thread, e: Throwable): Unit = {
+      println("Uncaught exception:" + e)
```
Comment (qiuxin2012) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala:42]: throw e， not println

Comment (ManfeiBai) [reply zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala:42]: Thanks! we would fix it soon~

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala [hunk 3]
```diff
@@ -67,10 +102,35 @@ class PyTorchModelSpec extends FunSuite with Matchers with BeforeAndAfterAll
...
   }
 
-  test("pytorch model can do predict") {
+  "PyTorch Model" should "do predict" in {
+    ifskipTest()
+    beforeAll()
+
     val inputTensor = Tensor[Float](1, 3, 224, 224).rand()
+    model.doLoadPyTorch(modelPath)
+    model2.doLoadPyTorch(modelPathMulti)
```
Comment (qiuxin2012) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/inference/PyTorchModelSpec.scala:133]: modelPath is resnet18, modelPathMulti is resnet50...
So how `r2 should be (results)` below.

**Comment by qiuxin2012** [comment]
jenkins passed

**Pull request merged**